opencog/moses/main/demo-problems.cc

```diff
@@ -261,20 +261,20 @@ void polynomial_problem::run(option_base* ob)
 	if (pms.exemplars.empty()) {
 		pms.exemplars.push_back(type_to_exemplar(id::contin_type));
 	}
+	type_tree tt = gen_signature(id::contin_type, 1);
+	ITable it(tt, (pms.nsamples>0 ? pms.nsamples : pms.default_nsamples));
+	type_node_seq tt_seq = type_tree_to_tyn_seq(tt);
```

Comment:
We should convert `tt` to `type_node_seq` only if `atomese_port` is on.

Reply:
Good point. I haven't noticed that!